tools/commands/build.py

```diff
@@ -159,12 +175,15 @@ def make(self):
         # may use pre-built object files while building
         self.clean()
 
+
 class mapbuild(object):
     def __init__(self, args):
-        if args.verbose: logger.setLevel(logging.DEBUG)
+        if args.verbose:
+            logger.setLevel(logging.DEBUG)
 
         commands = args.commands
-        _dep_modules = dep_modules if 'all' in args.modules or 'dep' in args.modules else [m for m in dep_modules if m in args.modules]
+        _dep_modules = dep_modules if 'all' in args.modules or 'dep' in args.modules else [
+            m for m in dep_modules if m in args.modules]
```

Comment:
I don't think this is a nice way to split. Better split on the else:

```suggestion
        _dep_modules = dep_modules if 'all' in args.modules or 'dep' in args.modules 
            else [m for m in dep_modules if m in args.modules]
```